.github/workflows/test_and_deploy.yml

```diff
@@ -35,18 +35,28 @@ jobs:
 
       - uses: tlambert03/setup-qt-libs@v1
```

Comment:
The test is geared towards setting up Qt libraries and test napari.
For example: [setup-qt-libs@v1](https://github.com/tlambert03/setup-qt-libs)

line: 41 also starts a display server... I sit possible to move this for 'plugin' only?

Reply:
Yep, I'll move that to a later point.